### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Goldinium**
added index.html
added style.css
added test.md to .gitignore file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,22 +2,17 @@
 
 # Project Name
 
-> One paragraph statement about the project.
+> Hello Microverse project
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
```
Comment (carlos-ssh) [inline README.md:9]: **- Kindly fill this information:**
Screen Shot 2021-08-10 at 06 03 10

_You are missing a few steps in your README file._

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (carlos-ssh) [inline .gitignore:1]: _**[OPTIONAL]**_
**- Add any other external files into your .gitignore, like:** .VS_Code, .idea, .DS_Store or node_modules.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Goldinium , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Add external files into your gitignore file.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (Emmyn5600) [reply .gitignore:1]: - [ ] Kindly add the  external files as requested by the previous code reviewer, check the above.

File: README.md (hunk 1 above)
Comment (Emmyn5600) [reply README.md:9]: - [ ] Kindly fix this issue by showing us how to get started with your project, please remove deployment under the Get started header and organize steps to follow clear.

File: .gitignore (hunk 2 above)
Comment (Emmyn5600) [reply .gitignore:1]: - [ ] Kindly add the  external files as requested by the previous code reviewer, check the above.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED :recycle:

Hi, @Goldinium 

Great work on implementing the changes requested by the previous reviewer, however there are some issues to be solved before you get approved but you are almost there:muscle:. 

Changes required :writing_hand:  
   
     * #### Please check the comments under the review too
     
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by carlos-ssh — APPROVED** [review]
Hi @Goldinium , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**